codex-rs/tui/src/conversation_history_widget.rs

```diff
@@ -358,7 +444,7 @@ impl WidgetRef for ConversationHistoryWidget {
                 // in the underlying buffer cells.  That means if a coloured line (for example a
                 // background task notification that we render in blue) happens to be underneath
                 // the scrollbar, the track and thumb adopt that colour and the scrollbar appears
-                // to “change colour”.  Explicitly setting the *track* and *thumb* styles ensures
+                // to "change colour".  Explicitly setting the *track* and *thumb* styles ensures
```

Comment:
Currently, comments are in US English throughout, so please maintain the existing style.